hazelcast-spring/src/main/resources/hazelcast-spring-3.6.xsd

```diff
@@ -275,16 +275,29 @@
                                         </xs:documentation>
                                     </xs:annotation>
                                 </xs:attribute>
-                                <xs:attribute name="optimize-queries" use="optional" type="parameterized-boolean"
-                                              default="false">
+                                <xs:attribute name="optimize-queries" use="optional" type="parameterized-boolean">
                                     <xs:annotation>
                                         <xs:documentation>
+                                            This parameter is deprecated as of Hazelcast 3.6
+                                            Use cache-deserialized-values attribute instead.
+
                                             This parameter is used to increase the speed of query processes in the map.
                                             It only works when `in-memory-format` is set as `BINARY` and performs
                                             a pre-caching on the entries queried.
                                         </xs:documentation>
```

Comment:
Maybe we should add a comment somewhere what happens if both options are used at once?